cpp/include/Ice/StreamHelpers.h

```diff
@@ -364,8 +364,10 @@ namespace Ice
     {
         static void write(OutputStream* stream, const T& v) { stream->writeException(v); }
 
-        // no read: only used for marshaling
-        // no print: we only print fields and a field can't be a user exception
+        // no read: we don't use this helper for unmarshaling.
+
+        // provide print for consistency even though there is no user exception field.
```

Comment:
I think this wording is slightly off.
`even though there is no user exception field` makes it sound like exceptions cannot have fields.
When what this is trying to convey is "exceptions cannot be stored in fields".

Maybe something like:
```suggestion
        // provide print for consistency even user exceptions cannot appear in fields.
```